SQL/Archive/2018-05-28_publication_schema.sql

```diff
@@ -0,0 +1,160 @@
+SET FOREIGN_KEY_CHECKS=0;
+-- Publication Status
+DROP TABLE IF EXISTS publication_status;
+CREATE TABLE `publication_status` (
+  `PublicationStatusID` int(2) unsigned NOT NULL,
+  `Label` varchar(255) NOT NULL,
+  PRIMARY KEY(`PublicationStatusID`)
+) ENGINE=InnoDB DEFAULT CHARSET='utf8mb4';
+DELETE FROM publication_status;
+INSERT INTO publication_status VALUES (1, 'Pending');
+INSERT INTO publication_status VALUES (2, 'Approved');
+INSERT INTO publication_status VALUES (3, 'Rejected');
+
+-- Main table
+DROP TABLE IF EXISTS publication;
+CREATE TABLE `publication` (
+    `PublicationID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+    `PublicationStatusID` int(2) unsigned NOT NULL default 1,
+    `UserID` int(10) unsigned NOT NULL,
+    `RatedBy` int(10) unsigned,
+    `DateProposed` date NOT NULL,
+    `DateRated` date default NULL,
+    `Title` varchar(255) NOT NULL,
+    `LeadInvestigator` varchar(255) NOT NULL,
+    `LeadInvestigatorEmail` varchar(255) NOT NULL,
+    `RejectedReason` varchar(255) default NULL,
+    `Description` text NOT NULL,
```

Comment:
Should probably include a DOI field for the publication itself.

Separately from that, a publication_doi table (which includes DOIs used/referenced by the publication) might be worth adding in the future, but the publication DOI for the publication itself (once published) is probably more important/easier to add.